src/engine/renderer/tr_shade.cpp

```diff
@@ -2325,52 +2325,43 @@ void Render_liquid( shaderStage_t *pStage )
 	GL_CheckErrors();
 }
 
-static void Render_fog()
+void Render_fog( shaderStage_t* pStage )
 {
-	fog_t  *fog;
-	float  eyeT;
-	vec3_t local;
-	vec4_t fogDistanceVector, fogDepthVector;
-
-	// no fog pass in snooper
-	if ( tess.surfaceShader->noFog || !r_wolfFog->integer )
-	{
+	if ( materialSystem.generatingWorldCommandBuffer ) {
+		Tess_DrawElements();
 		return;
 	}
 
-	// ydnar: no world, no fogging
-	if ( backEnd.refdef.rdflags & RDF_NOWORLDMODEL )
+	// no fog pass in snooper
+	// WTF is a snooper?
```

Comment:
This I guess: https://wolfenstein.fandom.com/wiki/Snooper_rifle

Reply:
Perhaps. Apparently Wolf:ET had a check for `tr.refdef.rdflags & RDF_SNOOPERVIEW` at https://github.com/id-Software/Enemy-Territory/blob/40342a9e3690cb5b627a433d4d5cbf30e3c57698/src/renderer/tr_shade.c#L803.

Reply:
I've removed that comment now because it doesn't apply to daemon.